integrationtests/python_src/px4_it/mavros/mission_test.py

```diff
@@ -48,18 +48,51 @@
 import sys
 import os
 import time
+import glob
+import json
 
 import mavros
 from pymavlink import mavutil
 from mavros import mavlink
 
+import px4tools
+
 from geometry_msgs.msg import PoseStamped
 from mavros_msgs.srv import CommandLong, WaypointPush
 from mavros_msgs.msg import Mavlink, Waypoint, ExtendedState
 from sensor_msgs.msg import NavSatFix
 from mavros.mission import QGroundControlWP
 #from px4_test_helper import PX4TestHelper
 
+def get_last_log():
+    try:
+        log_path = os.environ['PX4_LOG_DIR']
+    except KeyError:
+        log_path = os.path.join(os.environ['HOME'], 'ros/rootfs/fs/microsd/log')
+    last_log_dir = sorted(
+        glob.glob(os.path.join(log_path, '*')))[-1]
+    last_log = sorted(glob.glob(os.path.join(last_log_dir, '*.ulg')))[-1]
+    return last_log
+
```

Comment:
This functions converts the new qgc waypoint files to mavros waypoint data structures.